packages/server/src/instances/clearing/service.ts

```diff
@@ -34,6 +34,11 @@ export class InstanceClearingService extends Service {
 
     for (const channel of this.channels.list()) {
       channel.stateManager({
+        scopes: () =>
+          this.statesCache
+            .keys()
+            .filter((x) => x.startsWith(channel.meta.id))
+            .map((x) => this.statesCache.getValues(x)[1]),
```

Comment:
Maybe we should consider adding something better than `any` as the typing for the `statesCache`. That way, it would also make more sense what you access here!